site/content/vendir/docs/develop/vendir-spec.md

```diff
@@ -14,6 +14,11 @@ directories:
 - # path is relative to `vendir` CLI working directory
   path: config/_ytt_lib
 
+  # set the permission for this directory (optional; v0.32.2+)
```

Comment:
Not sure about the version, but i think the next one to be released is `v0.32.2`, right?

Reply:
Bumped it to `v0.32.3`, as `v0.32.2` has been release meanwhile.